testing/test_full.py

```diff
@@ -3193,8 +3194,91 @@ def test_generic_data_split():
     print("passed")
 
 
+def test_upload_download_huggingface(device):
```

Comment:
Updating test name to ensure it gets picked in the unit tests [[ref](https://github.com/mlcommons/GaNDLF/blob/master/.github/workflows/python-test.yml)]:
```suggestion
def test_train_inference_segmentation_histology_2d_huggingface(device):
```